modules/custom/falcon_development/config/install/features.bundle.falcon_feature.yml

```diff
@@ -1,100 +0,0 @@
-uuid: 17ba39ba-68d5-4c6b-8f8d-5316fd8fd699
```

Comment:
I'm afraid we can't do it - see for example https://github.com/systemseed/falcon/pull/52/files. So we need to figure out how to have this config imported properly.

Reply:
Yeah... we definitely need to have it :)